Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -382,80 +165,92 @@ void Server::start()
     
     for (size_t i = 0; i < threadsCount; ++i)
     {
-        mThreads.emplace_back(std::thread([this]() { mContext.run(); }));
+        _threads.emplace_back(std::thread([this]() { _context.run(); }));
     }
-    mPostgreManager->handleRequests();
+    _postgreManager->handleRequests();
 
-    Base::Logger::FileLogger::getInstance().log("[SERVER] Started!", Base::Logger::LogLevel::INFO);
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "[SERVER] Started!", 
+        Base::Logger::LogLevel::INFO
+    );
 }
 
 void Server::stop()
 {
-    mContext.stop();
+    _context.stop();
 
-    mPostgreManager->stopHandler();
+    _postgreManager->stopHandler();
 
-    for (std::thread& thread : mThreads)
+    for (std::thread& thread : _threads)
     {
         if (thread.joinable())
         {
             thread.join();
         }
     }
 
-    mThreads.clear();
-    Base::Logger::FileLogger::getInstance().log("[SERVER] Stopped!", Base::Logger::LogLevel::INFO);
+    _threads.clear();
+
+    Base::Logger::FileLogger::getInstance().log
+    (
+        "[SERVER] Stopped!",
+        Base::Logger::LogLevel::INFO
+    );
 }
 
 void Server::waitForClientConnection()
+{   
+    _acceptor.async_accept([this](const std::error_code error, asio::ip::tcp::socket socket) { acceptingClientConnection(error,socket); });
```

Comment:
Why you didn't create a separate functor variable and not set it like argument value?